core/src/main/java/io/questdb/std/Decimal256.java

```diff
@@ -0,0 +1,1764 @@
+                (p33 & 0xFFFFFFFFL) + (p42 & 0xFFFFFFFFL) + (p51 & 0xFFFFFFFFL) + (p60 & 0xFFFFFFFFL);
+        long r7 = (r6 >>> 32) + (p33 >>> 32) + (p42 >>> 32) + (p51 >>> 32) + (p60 >>> 32) +
+                (p43 & 0xFFFFFFFFL) + (p52 & 0xFFFFFFFFL) + (p61 & 0xFFFFFFFFL) + (p70 & 0xFFFFFFFFL);
+
+        this.ll = (r0 & 0xFFFFFFFFL) | ((r1 & 0xFFFFFFFFL) << 32);
+        this.lh = (r2 & 0xFFFFFFFFL) | ((r3 & 0xFFFFFFFFL) << 32);
+        this.hl = (r4 & 0xFFFFFFFFL) | ((r5 & 0xFFFFFFFFL) << 32);
+        this.hh = (r6 & 0xFFFFFFFFL) | ((r7 & 0xFFFFFFFFL) << 32);
+    }
+
+    private void multiply192(long h, long m, long l) {
+        // Perform 256-bit × 192-bit multiplication
+        // Result is at most 448 bits, but we keep only the lower 256 bits
+
+        // Split this into eight 32-bit parts
+        long a7 = hh >>> 32;
+        long a6 = hh & 0xFFFFFFFFL;
+        long a5 = hl >>> 32;
+        long a4 = hl & 0xFFFFFFFFL;
+        long a3 = lh >>> 32;
+        long a2 = lh & 0xFFFFFFFFL;
+        long a1 = ll >>> 32;
+        long a0 = ll & 0xFFFFFFFFL;
+
+        long b5 = h >>> 32;
+        long b4 = h & 0xFFFFFFFFL;
+        long b3 = m >>> 32;
+        long b2 = m & 0xFFFFFFFFL;
+        long b1 = l >>> 32;
+        long b0 = l & 0xFFFFFFFFL;
+
+        // Compute all partial products
+        long p00 = a0 * b0;
+        long p01 = a0 * b1;
+        long p02 = a0 * b2;
+        long p03 = a0 * b3;
+        long p04 = a0 * b4;
+        long p05 = a0 * b5;
+        long p10 = a1 * b0;
+        long p11 = a1 * b1;
+        long p12 = a1 * b2;
+        long p13 = a1 * b3;
+        long p14 = a1 * b4;
+        long p15 = a1 * b5;
+        long p20 = a2 * b0;
+        long p21 = a2 * b1;
+        long p22 = a2 * b2;
+        long p23 = a2 * b3;
+        long p24 = a2 * b4;
+        long p25 = a2 * b5;
+        long p30 = a3 * b0;
+        long p31 = a3 * b1;
+        long p32 = a3 * b2;
+        long p33 = a3 * b3;
+        long p34 = a3 * b4;
+        long p35 = a3 * b5;
+        long p40 = a4 * b0;
+        long p41 = a4 * b1;
+        long p42 = a4 * b2;
+        long p43 = a4 * b3;
+        long p44 = a4 * b4;
+        long p45 = a4 * b5;
+        long p50 = a5 * b0;
+        long p51 = a5 * b1;
+        long p52 = a5 * b2;
+        long p53 = a5 * b3;
+        long p54 = a5 * b4;
+        long p55 = a5 * b5;
+        long p60 = a6 * b0;
+        long p61 = a6 * b1;
+        long p62 = a6 * b2;
+        long p63 = a6 * b3;
+        long p64 = a6 * b4;
+        long p65 = a6 * b5;
+        long p70 = a7 * b0;
+        long p71 = a7 * b1;
+        long p72 = a7 * b2;
+        long p73 = a7 * b3;
+        long p74 = a7 * b4;
+        long p75 = a7 * b5;
+
+        long overflow = p35 | p44 | p45 | p53 | p54 | p55 | p62 | p63 | p64 | p65 | p71 | p72 | p73 | p74 | p75;
+        if (overflow != 0) {
+            throw NumericException.instance().put("Overflow");
```

Comment:
it would be good to add codes to "overflow" text, in case exception is logged, we can identify where is was thrown from